src/components/common/animations/Error/index.js

```diff
@@ -17,9 +17,11 @@ class Error extends AnimationBase {
         source={this.improveAnimationData(animationData)}
         autoSize={false}
         style={{
+          top: -getDesignRelativeHeight(97, false) * 0.33,
```

Comment:
please add comment about what this "voodoo" is doing?
have you tested it on different screen sizes?

Reply:
also please move outside and use HoC `withStyles`

Reply:
fixed. I've also added more screenshots to the description to have different screen sizes reference